react/InfosCarrousel/styles.styl

```diff
@@ -14,24 +14,26 @@
 
 .InfosCarrousel-navigation
     position absolute
-    bottom rem(10)
-    right spacing_values.xs
+    bottom (spacing_values.xs)
+    right (spacing_values.xs)
     display flex
     align-items center
 
     +mobile()
         bottom rem(4)
-        right 0
+        right rem(4)
```

Comment:
I don't think we want the extra offset on mobile?

Reply:
Seen in Zoom